src/components/auth/socialAuth.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import FacebookLogin from "react-facebook-login";
+import GoogleLogin from "react-google-login";
+import { connect } from "react-redux";
+import { facebookAuth, googleAuth } from "../../actions/authActions/socialAuth";
+import { Redirect } from "react-router-dom";
+import PropTypes from "prop-types";
+
+export class SocialAuth extends Component {
+  onFailure = error => {
+    alert(error);
+  };
+  responseGoogle = response => {
+    this.props.googleAuth({ access_token: response.accessToken });
+  };
+
+  responseFacebook = response => {
+    this.props.facebookAuth({ access_token: response.accessToken });
+  };
+  render() {
+    if (this.props.isLoggedIn) {
+      return <Redirect to="/" />;
+    }
+    return (
+      <div className="social-btn text-center mission">
+        <FacebookLogin
+          appId="2121134901264753"
```

Comment:
Replace the appId value with an environment variable so that such sensitive data isn't exposed

Reply:
@LehruAngela  I have made this change